.github/workflows/pgspot.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
         --proc-without-search-path
           'extschema.cagg_migrate(_cagg regclass,_override boolean = FALSE,_drop_old boolean = FALSE)'
```

Comment:
shouldn't the default values be removed from `cagg_migrate` too?

Reply:
oh cagg_migrate has 2 entries, i think the first one is no longer needed